webapp/components/_new/features/MySomethingList/MySomethingList.vue

```diff
@@ -155,8 +145,42 @@ export default {
       this.editingItem = null
       this.disabled = true
     },
-    async handleDeleteItem(item) {
-      await this.callbacks.delete(this, item)
+    handleDeleteItem(item) {
+      this.openModal(item)
+    },
+    openModal(item) {
+      this.commitModalData(this.modalData(item))
+    },
+    modalData(item) {
```

Comment:
computed?

Reply:
This can not be a comuted, because it needs the item parameter which is not globally defined.
Or? @Mogge